packages/amplify-graphql-types-generator/test/__snapshots__/loading.ts.snap

```diff
@@ -384,6 +384,6 @@ Object {
 `;
 
 exports[`Validation should throw a helpful message when a file has invalid gql snippets 1`] = `
-"Could not parse graphql operations in test/fixtures/misc/invalid-gqlQueries.js
+"Could not parse graphql operations in packages/amplify-graphql-types-generator/test/fixtures/misc/invalid-gqlQueries.js
```

Comment:
why is this here?

Reply:
Yeah this needs to be reverted. It will cause graphql tests to fail